### PR TITLE
Update for moveit PR #1724: Jog Arm Dimensions

### PR DESCRIPTION
**Pull request by AdamPettinger**
See more here: https://github.com/ros-planning/moveit/pull/1724

### Description

Please explain the changes you made, including a reference to the related issue if applicable

### Checklist
- [ ] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] While waiting for someone to review your request, please consider reviewing [another open pull request](https://github.com/ros-planning/moveit/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

**Comment by davetcoleman** [comment]
Thanks! Should be merged with https://github.com/ros-planning/moveit/pull/1724#pullrequestreview-322369641

File: doc/arm_jogging/arm_jogging_tutorial.rst [hunk 1]
```diff
@@ -76,6 +76,10 @@ Settings
 --------
 User-configurable settings of the jog node are well-documented in ``jog_arm/config/ur_simulated_config.yaml``.
 
+Changing Control Dimensions
+---------------------------
+The dimensions jogging is allowed in may be changed with the ``change_control_dimensions`` (``moveit_msgs/ChangeControlDimensions``) service. The service consists of the 6 booleans representing the manipulator's degrees of freedom (3 translation and 3 rotation), defined in the input velocity command frame. Setting one of these booleans to ``false`` will result in the user input in that direction being overwritten with a 0, thus disallowing motion in the direction. 
+
```
Comment (davetcoleman) [inline doc/arm_jogging/arm_jogging_tutorial.rst]: is there a tool for using this service? like RQT-based GUI?

Comment (AdamPettinger) [reply doc/arm_jogging/arm_jogging_tutorial.rst]: I am currently using this service in a QT GUI that includes functionality for our specific robot/task. 

I could possibly separate it into its own QT GUI, or create an RQT GUI for it

Comment (davetcoleman) [reply doc/arm_jogging/arm_jogging_tutorial.rst]: if there isn't an easy out of the box suggestion to add to the tutorial, no problem. i was thinking this was more like dynamic reconfigure, but its not.

**Comment by davetcoleman** [comment]
let's merge after https://github.com/ros-planning/moveit/pull/1724 is merged.

**Comment by AdamPettinger** [comment]
Sorry, was not trying to close and reopen haha

[ros-planning/moveit#1724](https://github.com/ros-planning/moveit/pull/1724) is merged now

**Pull request merged**